features/mfa-recovery.feature

```diff
@@ -17,23 +17,19 @@ Feature: Send a code to an MFA recovery contact
     Then I should see a way to send an MFA recovery code to my manager
     And I should see a way to send an MFA recovery code to another recovery contact
 
-  Scenario: User with manager and recovery contact, send code to recovery contact
+  Scenario Outline: User with manager and recovery contact, send code to each
     Given I use an IDP that is configured to offer MFA recovery-contacts
     And I provide credentials that have backup codes and a manager
     And I log in
     And I click the Request Assistance link
-    When I send the code to the recovery contact
+    When I send the code to the <recipient>
     Then I should not see an error message
     And I should see confirmation that the code was sent
```

Comment:
Is there a way of confirming it was sent to the correct person?

Reply:
Not yet. I haven't thought of a simple/clean way to do that. If this test was actually making calls to ID Broker, I could probably call the `GET /user/{employee_id}/mfa` ID Broker API endpoint and confirm that the correct entry was added. (It's currently calling a FakeIdBrokerClient.)

I'll add a note to try to do that later.

Reply:
Clarification: I plan to do that in a later PR, not in this one.